### PR TITLE
Changes electropacks to be a child of /signaler/

### PR DESCRIPTION
**Pull request by UndeadNappist**
This is mostly to make a point about the hierarchy system and copy-pasted code, along with some mild optimization I was playing with.

This is derived from how the codebase is disorganized.

### CODE REVIEW

**Comment by PKPenguin321** [comment]
I think this will make it attachable to shit like flashes if you screwdriver it which will probably be really weird

**Comment by UndeadNappist** [comment]
It does. It doesn't do anything, and makes a defunct assembly that does nothing, although in the future if someone wanted to they could merge the helmet construction to use the same procs.

It could also open up the door for having timer or proximity based electropacks instead of having it be a separate device. Or you could make the electropack an empty box that contains a signaler and an igniter and some wires, etc.

It doesn't really matter.

**Comment by JJRcop** [comment]
screenshot_20190301_210408

**Comment by JJRcop** [comment]
screenshot_20190301_210504

**Comment by Tlaltecuhtli** [comment]
was this tested

**Comment by optimumtact** [comment]
your point is lost in the fact you can't even use git properly

**Comment by UndeadNappist** [comment]
> was this tested

Works on another server where this change has already been implemented.

>your point is lost in the fact you can't even use git properly

and you add pointless nagware logs when there's already too many of them, what's your point?

**Comment by JJRcop** [comment]
BadDeathclaw/TG-Claw#614 for reference

File: code/modules/assembly/electropack.dm [hunk 1]
```diff
@@ -11,32 +11,25 @@
 	w_class = WEIGHT_CLASS_HUGE
 	materials = list(MAT_METAL=10000, MAT_GLASS=2500)
 	var/on = TRUE
-	var/code = 2
-	var/frequency = FREQ_ELECTROPACK
 	var/shock_cooldown = 0
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/assembly/electropack.dm]: new variables go at the bottom

File: code/modules/assembly/shock_kit.dm [hunk 2]
```diff
@@ -4,7 +4,7 @@
 	icon = 'icons/obj/assemblies.dmi'
 	icon_state = "shock_kit"
 	var/obj/item/clothing/head/helmet/part1 = null
-	var/obj/item/electropack/part2 = null
+	var/obj/item/assembly/signaler/electropack/part2 = null
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/assembly/shock_kit.dm]: remove both `= null`s here

Comment (ExcessiveUseOfCobblestone) [reply code/modules/assembly/shock_kit.dm]: also new vars go at the bottom

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]

**Comment by PKPenguin321** [comment]
Also im pretty sure you wouldn't be able to equip assembly-fied electropacks which kind of defeats the point

**Pull request closed without merging**